projects/experimental/directives/card/card.styles.less

```diff
@@ -31,4 +32,83 @@
     [tuiSubtitle] {
         max-width: 100%;
     }
+
+    &[data-size='l'] {
+        --t-size: 100%;
+        height: auto;
+        background: var(--tui-base-01);
+        border-radius: var(--t-radius);
+        padding: var(--t-padding);
+        gap: var(--t-gap);
+        box-shadow: 0 0.375rem 2.125rem 0 rgba(0, 0, 0, 0.12);
+        font-size: 1rem;
+
+        & > * {
+            width: 100%;
+        }
+
+        [tuiCell] {
+            --t-padding: 0.5rem 0;
+            box-sizing: border-box;
+        }
+
+        &[data-appearance='normal'] {
+            --t-gap: 1.25rem;
+            --t-radius: 1.5rem;
+            --t-padding: 1rem 1.5rem;
+
+            & > *:first-child:not(:last-child) {
+                margin-top: 0.25rem;
+            }
+
+            & > * + *:last-child {
+                margin-bottom: 0.5rem;
+            }
+        }
+
+        &[data-appearance='compact'] {
+            --t-gap: 1rem;
+            --t-radius: 1rem;
+            --t-padding: 1rem 1.25rem;
```

Comment:
If vertical padding is always `1rem` we should probably update this value:
https://github.com/taiga-family/taiga-ui/blob/main/projects/experimental/directives/surface/surface.style.less#L12

Reply:
fixed